tests/datalab/test_cleanvision_integration.py

```diff
@@ -0,0 +1,200 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+import pandas as pd
+
+from cleanlab import Datalab
+
+LABEL_NAME = "label"
+IMAGE_NAME = "image"
+IMAGELAB_ISSUE_TYPES = [
+    "dark",
+    "light",
+    "low_information",
+    "odd_aspect_ratio",
+    "odd_size",
+    "grayscale",
+    "blurry",
+]
+SEED = 42
+
+
+class TestCleanvisionIntegration:
+    @pytest.fixture
+    def features(self, image_dataset):
+        np.random.seed(SEED)
+        return np.random.rand(len(image_dataset), 5)
+
+    @pytest.fixture
+    def pred_probs(self, image_dataset):
+        np.random.seed(SEED)
+        return np.random.rand(len(image_dataset), 2)
+
+    @pytest.fixture
+    def set_plt_show(self, monkeypatch):
+        monkeypatch.setattr(plt, "show", lambda: None)
+
+    @pytest.mark.usefixtures("set_plt_show")
+    def test_imagelab_issues_checked(self, image_dataset, pred_probs, features, capsys):
+        datalab = Datalab(data=image_dataset, label_name=LABEL_NAME, image_key=IMAGE_NAME)
+        datalab.find_issues(pred_probs=pred_probs, features=features)
+        captured = capsys.readouterr()
+        assert (
+            "Finding dark, light, low_information, odd_aspect_ratio, odd_size, grayscale, blurry images"
+            in captured.out
+        )
+        # unable to check for non iid as feature space is too small, skipping it in interest of time
+        assert "Failed to check for these issue types: [NonIIDIssueManager]" in captured.out
+        assert len(datalab.issues) == len(image_dataset)
+        # 14 from imagelab + 6 from datalab
+        assert len(datalab.issues.columns) == 14 + 6
+        assert len(datalab.issue_summary) == 7 + 3
```

Comment:
where do these magic numbers come from? are they forwards compatible as more issue types are added in datalab? 

At least define them as variables with explanation in a comment, so if the unit test breaks in the future when somebody adds new datalab code, they can more easily update the unit test

Reply:
Added variables for these numbers